app/components/developer-program.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+
+
+//  I M P O R T
+
+import html from "choo/html";
+
+
+
+//  E X P O R T
+
+export default () => html`
+  <developer-program>
+    <p>If this sounds like you, <a href="#" class="no-smooth" id="get-started">get started here</a>!</p>
```

Comment:
Can this be a single centered button labeled "Claim Developer LBC" with smaller/meta text below it that says "This will authenticate you with GitHub to prove eligibility as well as mark you as a follower of LBRY".

Reply:
I was going to change this myself, but we don't have a button style?!